paths/paths.go

```diff
@@ -210,6 +211,15 @@ var drivePrefixes = map[Drive]prefix{
 	TestDrive:         {"./testdata/", "./testdata/", "/dev/null/"},
 }
 
+func SafeParse(ctx workflow.Context, path string) (Path, error) {
```

Comment:
Maybe this shouldnt be in paths, but rather in wfutils or another place.
Or rename ParseSideEffect() or something like that.